src/compiler/commandLineParser.ts

```diff
@@ -536,21 +589,24 @@ namespace ts {
       *    file to. e.g. outDir
       */
     export function parseJsonConfigFileContent(json: any, host: ParseConfigHost, basePath: string, existingOptions: CompilerOptions = {}, configFileName?: string): ParsedCommandLine {

```

Comment:
the `configFileName` parameter is optional here, but it is required in local functions inside `parseJsonConfigFileContent`, would that lead to confusion?